src/app/initialisation/syncPoolService.js

```diff
@@ -49,9 +49,9 @@ function SyncPoolService($q, mediator, syncService) {
 
     //Initialisation of sync data sets to manage.
     return $q.all([
-      syncService.manage(config.datasetIds.workorders, {}, {filter: filter}, config.syncOptions),
-      syncService.manage(config.datasetIds.workflows, {}, {}, config.syncOptions),
-      syncService.manage(config.datasetIds.results, {}, {}, config.syncOptions)
+      syncService.manage(config.datasetIds.workorders, config.syncOptions.workorders, {filter: filter}, {}),
```

Comment:
To provide top level configurability, hiding this initiation for sync config could obfuscate it/move it too far from the user. Its worth looking at better implementation tough.